WEB_GUIDE.md

```diff
@@ -466,17 +468,153 @@ callback("Error message");
+
+You have the ability to control the direct deep linking of each link as well:
+
+| Key | Value
+| --- | ---
+| "$deeplink_path" | The value of the deep link path that you'd like us to append to your URI. For example, you could specify "$deeplink_path": "radio/station/456" and we'll open the app with the URI "yourapp://radio/station/456?link_click_id=branch-identifier". This is primarily for supporting legacy deep linking infrastructure.
+
+#### Usage
+```js
+branch.deepview(
+    data,
+    options,
+    callback (err)
+);
+```
+
+#### Example
+```js
+branch.deepview(
+    {
+        channel: 'facebook',
+        data: {
+            mydata: 'content of my data',
+            foo: 'bar',
+            '$desktop_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$ios_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$ipad_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$android_url': 'https://en.wikipedia.org/wiki/Internet',
+            '$deepview_path': 'item_id=12345',
+            '$og_app_id': '12345',
+            '$og_title': 'My App',
+            '$og_description': 'My app\'s description.'
+        },
+        feature: 'dashboard',
+        stage: 'new user',
+        tags: [ 'tag1', 'tag2' ],
+    },
+    {
+        make_new_link: true,
+        open_app: true
+    },
+    function(err) {
+        console.log(err || 'no error');
+    }
+);
+```
+
+##### Callback Format
+```js
+callback(
+    "Error message"
+);
+```
+
+
+
+### Branch.deepviewCta
+
+Perform the branch deepview CTA (call to action) on mobile. Namely, depends on how
+*branch.deepview* is set up, the mobile users are redirected accordingly. If the deepview is
+configured with the option *`open_app`* being true, an immediate attempt is made as soon as
+deepview finishes, and thus the CTA is to visit the platform-appropriate app stores; if on the
+other hand, the deepview is configured with the option *`open_app`* being false, the CTA is to
+try to open app, and to visit the platform-appropriate app stores if the open-app attempt failes.
+
+If *branch.deepview* has not been called, an error will arise with a reminder to call
+*branch.deepview* first.
+
+##### Usage
+```js
+&#x2F;&#x2F; If you&#39;re using jQuery, then:
+$(&#39;a.deepview-cta&#39;).click(branch.deepviewCta);
+
+&#x2F;&#x2F; Or you can:
+document.getElementById(&#39;my-elem&#39;).onClick = branch.deepviewCta;
+
+&#x2F;&#x2F; Or in HTML you can:
+&lt;a href=&#39;...&#39; onclick=&#39;branch.deepviewCta()&#39;&gt;
+
+&#x2F;&#x2F; If you wish to dedicate a CTA link only to branch deepviewCta, you can:
+branch.deepview(data, option, function(err) {
+    if (err) {
+        throw err;
+    }
+    ${&#39;a.deepview-cta&#39;).click(branch.deepviewCta);
+});
+
+&#x2F;&#x2F; You can call this function any time after branch.deepview() is finished by simply:
+branch.deepviewCta();
```

Comment:
If I'm a dev, I would simply make all my buttons call deepviewCta. If there is an init blip that causes the web sdk to fail to load, at least the the bnc.lt link would still be formatted. I wouldn't know if the user action actually failed or not.